src/benchq/resource_estimation/_footprint_analysis.py

```diff
@@ -16,6 +16,7 @@
 import dataclasses
 import datetime
 import math
+import sys
```

Comment:
Just a note to remove the sys import in the case that you replace the sys.exit() with a python error as discussed [here](https://github.com/zapatacomputing/benchq/pull/106/files/6eace441a275d630e374a8b882d9bec3f85d1d69#r1273995425).

Reply:
Sure!